config.sh.hrp5p01v

```diff
@@ -0,0 +1,12 @@
+source config.sh.defaults
+
+######## Please edit from here ########
+export PREFIX=$HOME/usr
+export SUDO=
+export MAKE_THREADS_NUMBER=4
+export BUILD_TYPE=RelWithDebInfo
+export ROBOT_HOSTNAME=hrp5p01c.local
+export PACKAGES="OpenRTM-aist openhrp3 hrpsys-base HRP5P hmc2 hrpsys-humanoid"
```

Comment:
I think you can remove HRP5P, hmc2 and hrpsys-humanoid.
If this is correct, you can also remove the line no.8.

Reply:
Some rosbridge nodes are launched in hrp5p01v and HRP5P, hmc2 and hrpsys-humanoid is needed by them.
- HRP5P model is needed by some packages in ros (ex. aist_hrpsys_ros_bridge_tutorials, which converts vrml model into urdf model)
- idl files in hmc2 and hrpsys-humanoid is needed to generate ROSBridge in aist_hrpsys_ros_bridge.